opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/SpanConverter.java

```diff
@@ -40,10 +41,9 @@ static SpanKind mapKind(@Nullable io.opencensus.trace.Span.Kind ocKind) {
     return SpanKind.INTERNAL;
   }
 
-  @Nullable
   static Span fromOtelSpan(@Nullable io.opentelemetry.api.trace.Span otSpan) {
     if (otSpan == null) {
-      return null;
+      return BlankSpan.INSTANCE;
```

Comment:
this is a behavior change. Do we have a unit test for this?

Reply:
The oc shim has a lot of complication that makes it difficult for me to figure out some of the test cases - that's why it has very low total coverage already. Hoping someone from oc can give this artifact some attention.

I found this behavior in the OC scope manager so copied it since it seemed like a good way to reduce nullness.